addon/components/paper-dialog-inner.js

```diff
@@ -4,10 +4,10 @@ import PaperDialog from './paper-dialog';
 
 const { Component, computed } = Ember;
 
-export default Component.extend(Translate3dMixin, {
+export default Component.extend(/*Translate3dMixin, TODO */ {
   tagName: 'md-dialog',
-  classNames: ['md-default-theme'],
-  classNameBindings: ['contentOverflow:md-content-overflow', 'fullscreen:md-dialog-fullscreen'],
+  classNames: ['md-default-theme'     , 'md-transition-in'     /* TODO */ ],
+  classNameBindings: ['contentOverflow:md-content-overflow', 'dialogComponent.fullscreen:md-dialog-fullscreen'],
```

Comment:
We use dialogComponent in `paper-inner-dialog` for 4 different things. We could pass it by binding `this`, we could bind all four things we use, or we can keep what we have. I thought the discussion from `tabs` was that it was better to connect the two components than to bind a bunch of properties between two components.